Storage/tests/Unit/SigningHelperTest.php

```diff
@@ -81,7 +81,7 @@ public function testV2Sign()
     /**
```

Comment:
Can we please clean up the references to `cname` in this test? https://github.com/googleapis/google-cloud-php/blob/5a2a3287fb345a81ffa9edd5ffa92c5ed2df4e93/Storage/tests/Unit/SigningHelperTest.php#L603-L606